dll/win32/shell32/dialogs/dialogs.cpp

```diff
@@ -151,9 +151,8 @@ DoLoadIcons(HWND hwndDlg, PPICK_ICON_CONTEXT pIconContext, LPCWSTR pszFile)
         }
     }
 
-    // Set the text and reset the edit control's modification flag
```

Comment:
I would suggest you keep a simplified comment here:
```suggestion
    // Set the text and reset the selection to the first icon
```

Reply:
How is this not implied by `SetDlgItemTextW` and `LB_SETCURSEL`? What value does your comment add?